include/bouncer.h

```diff
@@ -192,6 +193,11 @@ extern int cf_sbuf_len;
  */
 #define MAX_PASSWORD    2048
 
+#ifdef HAVE_LDAP
+/* Hope this length is long enough for ldap config line */
+#define MAX_LDAP_CONFIG 1024
```

Comment:
Is it a reasonable assumption that 1024 is long enough? What exactly do we store in here?

Reply:
This line contents the content in one line of hba which starts after "ldap".
Take this line for example:
`host all ldapuser1 0.0.0.0/0 ldap ldapurl="ldap://127.0.0.1:389/dc=example,dc=net?uid?sub"`
The content will be:
`ldapurl="ldap://127.0.0.1:389/dc=example,dc=net?uid?sub"`
So I think 1024 should be large enough to hold all the content after ldap.